website/src/components/UserChoice.tsx

```diff
@@ -0,0 +1,51 @@
+import { Card, CardBody, Flex, Heading } from "@chakra-ui/react";
+import Link from "next/link";
+import React from "react";
+import Image from "next/image";
+
+const UserChoice = () => {
+  return (
+    <Flex gap={10} wrap="wrap" justifyContent="center">
```

Comment:
A plan I have is to have two sections: `create` and `evaluate` type tasks.  What about restructuring (and likely `Option` its own component) this a bit so it's a higher level component where someone can do the following:

```
<TaskOptions title="Create">
  <TaskOption title="Summarize Stories" link="..." icon={....} />
  ...
</TaskOptions>

<TaskOptions title="Evaluate">
  <TaskOption title="Rate Summaries" link="..." icon={....} />
  ...
</TaskOptions>
```

Reply:
Great idea! i shall rename the components and restructure them. This was a 4AM idea so my naming convention wont be on point, I'll have them done tomorrow. Also I wanted to suggest to maybe open more tasks for UI improvements? I would like to make more things pretty and also add a dark mode.

Reply:
Sure, I can do a sweep through the current state of things and start listing more opportunities.